addons/armaos/functions/fnc_terminal_addEventHandler.sqf

```diff
@@ -0,0 +1,76 @@
+// https://community.bistudio.com/wiki/DIK_KeyCodes
+#include "\a3\ui_f\hpp\definedikcodes.inc"
+
+params ["_terminalCtrl"];
+
+/* ---------------------------------------- */
+
+private _result = _terminalCtrl ctrlAddEventHandler
+[
+	"KeyDown", 
+	{
+		params ["_displayorcontrol", "_key", "_shift", "_ctrl", "_alt"];
+
+		//hint format ["OUTPUT CONTROL KEY: %1 \n SHIFT: %2 \n CTRL: %3 \n ALT: %4", _key call BIS_fnc_keyCode, _shift, _ctrl, _alt];
```

Comment:
Dead code

Reply:
removed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/bdeec0439c3a89ef1583c3b663196a37a6c51e0b